about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/d404a68fe3.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+    <title>About section</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="check-btn">
+            <i class="fa-solid fa-bars fa-2x" id="bars-icon" onclick="myFunction(this)"></i>
+        </a>
+        <nav class="navbarNav">
+            <a href="index.html" class="logo"><img src="assets/saldana-degisn.png" alt="logo"></a>
+            <div class="nav-options">
+                <ul class="nav-list">
+                    <li class="nav-items"><a href="about.html">About</a></li>
+                    <li class="nav-items"><a href="index.html#program">Program</a></li>
+                    <li class="nav-items"><a href="index.html#featured">Featured</a></li>
+                    <li class="nav-items"><a href="index.html#sponsor">Sponsor</a></li>
+                </ul>
+            </div>
+        </nav>
```

Comment:
## OPTIONAL 

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![nav](https://user-images.githubusercontent.com/69638013/158008396-25e033d2-090c-44d7-a7f3-97d8d94055db.png)  |  ![image](https://user-images.githubusercontent.com/69638013/158008390-b0cec151-f314-4245-ab31-b925bd93f58f.png)


- [ ] Kindly add the top dark navigation as this is part of the project requirements and it would make your design look like the  behance design 👍🏿  

- [ ] Kindly add more padding to the top and bottom of the white nav so the logo and last link has more spacing 👍🏿